app/controllers/shopify_app/callback_controller.rb

```diff
@@ -45,6 +73,10 @@ def associated_user
       auth_hash['extra']['associated_user']
     end
 
+    def associated_user_id
+      associated_user && associated_user['id']
+    end
```

Comment:
This seems to be on the edge of superfluous, but doing safe navigation with nil/hash objects seems sort of gnarly. Here's what I found:

```ruby
associated_user&.[]('id')
```

If that feels like readable Ruby, you could put that in at line 43 and not need this method, but I dunno...
```suggestion
    end
```
```suggestion
    end
```
```suggestion
    end
```

Reply:
It's clever but not idiomatic :)